### PR TITLE
Expose installAmpInaboxHost as a global function.

### PR DESCRIPTION
**Pull request by lannka**

### CODE REVIEW

File: ads/inabox/inabox-host.js [hunk 1]
```diff
@@ -24,12 +24,12 @@ import {dev, initLogConstructor} from '../../src/log';
 import {InaboxMessagingHost} from './inabox-messaging-host';
 
 const TAG = 'inabox-host';
-run(self);
+self.installAmpInaboxHost = installAmpInaboxHost;
```
Comment (zhouyx) [inline ads/inabox/inabox-host.js]: question: who will call the `#installAmpInaboxHost` now? ad script?

Comment (lannka) [inline ads/inabox/inabox-host.js]: yes. like in the example.

**Comment by zhouyx** [comment]
lgtm

File: ads/inabox/inabox-host.js (hunk 1 above)
Comment (bradfrizzell) [inline ads/inabox/inabox-host.js]: what is 'self' here?

Comment (lannka) [inline ads/inabox/inabox-host.js]: `self === window`
https://developer.mozilla.org/en-US/docs/Web/API/Window/self

**Comment by bradfrizzell** [comment]
I think we do not need this as a global function as per discussion with @keithwrightbos

**Pull request closed without merging**